### ISSUE TITLE
Conflicts with CRT macros in VS2015

### ISSUE THREAD

**Issue by speps** [p12tic/libsimdpp#43]
I tried to use libsimdpp in VS2015. However it resulted in errors because ucrt/stdlib.h defines min and max as macros so any occurence of min/max in libsimdpp was getting replaced.

Not sure what's the best way to fix that but currently I added `#undef` for both of those in null/math.h.

**Comment by p12tic** [comment]
Thanks for reporting. Your solution makes sense.

**Comment by alukutkin** [comment]
Common solution for that is to add NOMINMAX definition in cmake file:
```
if(MSVC)
    target_compile_definitions(${PROJECT_NAME} PUBLIC NOMINMAX)
endif()
```

**Comment by p12tic** [comment]
Fixed in c262b3d68c0504d33ed182417141e70789b5e5b0. Thanks!
